hazelcast/src/main/java/com/hazelcast/wan/impl/WanReplicationPublishersContainer.java

```diff
@@ -119,7 +122,7 @@ public void republishReplicationEvent(WanReplicationEvent wanReplicationEvent) {
 
     public void checkWanReplicationQueues() {
```

Comment:
maybe we can use `doPrepublicationChecks` to be consistent instead of `checkWanReplicationQueues`

Reply:
Right, will change.

Reply:
Addressed in a separate commit.